### PR TITLE
Add memory, cpu combined usage chart and storage usage bar

### PR DESCRIPTION
**Pull request by EricJeffrey**
image

Charts are implemented using `echarts`.  Data are fetched by `cockpit.series`, I'm not very sure about its mechanism, but from what I see, it looks good. These charts component has no state, it just updates itself by `cockpit.grid`.

Storage usage are implemented using `/libpod/system/df`, with sum of all `image.Size` and `container.Size`.  The storage usage bar fixed, cause really don't know how to change its length without knowing a max number. 
Still, I'll leave those tests to you if possible 😅.

### CODE REVIEW

**Comment by EricJeffrey** [comment]
trying to fix #165

**Comment by marusak** [comment]
Hmm, I cannot make this to work, I don't get any numbers.
noactivty

What version of podman you made this to work with?
It would be great if you could check with our testing VMs that it works. And then you could also write tests :) Let me know what is the problem you see and we can hopefully resolve that.

**Comment by EricJeffrey** [comment]
I'm working with podman2.0.2 on my ubuntu20. I'll try to run it on test server and see how it will go.

**Comment by EricJeffrey** [comment]
Ok, I'll try make it work on fedora first then fix these bugs 😂

File: src/client.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
+/* eslint-disable no-trailing-spaces */
```
[the remark this answers is not in the document]
Comment (EricJeffrey) [reply src/client.js:1]: this is a miss, not what I meant to be

**Comment by EricJeffrey** [comment]
Now block on [#14420](https://github.com/cockpit-project/cockpit/issues/14420), turn out to be something with CGroupV2.

**Comment by marusak** [comment]
> Now block on #14420, turn out to be something with CGroupV2.

`podman stats` works just fine, so you might want to check out how that works. Also it might be worth trying to contact podman team to get some advice or to see if they would not expose it on  API.

**Comment by EricJeffrey** [comment]
Problem is that the backend of cockpit cannot fetch these data on CGroupV2, but **can** on CGroupV1, so it differs on Ubuntu(CGroupV1) and Fedora(CGroupV2). 
Surely it would be ok if podman expose API, but as cockpit once gave a way for this, maybe they will fix the *bug* and make it work again?

**Comment by martinpitt** [comment]
Agreed that a podman resource API would be better from Cockpit's POV than making assumptions about how podman creates its cgroup structure (layer violation).

**Comment by EricJeffrey** [comment]
Yeah, not to say when user specify different CGroup manager or containers of k8s using podman as driver. 
So maybe for now I just fix storage here and remove these CPU and memory charts? what do you think?

**Comment by EricJeffrey** [comment]
Or maybe, we can make use of data from `podman stats` and put them into charts? Though may not work well.

**Comment by EricJeffrey** [comment]
Close as it won't work well.

**Pull request closed without merging**